app/src/main/assets/country_metadata/GB.yml

```diff
@@ -4,10 +4,10 @@ isAdvisorySpeedLimitKnown: true
 isLeftHandTraffic: true
 isLivingStreetKnown: true
 isSlowZoneKnown: true
+measurementSystem: [imperial, metric]
 mobileCountryCode: 234
 officialLanguages: [en]
 orchardProduces: [apple, strawberry, raspberry, pear, plum, cherry, hop, grape, tomatoe, chilli_pepper,
   sweet_pepper]
 popularReligions: [christian, muslim, jewish, hindu, sikh]
 popularSports: [soccer, tennis, bowls, cricket, golf, basketball, rugby]
-speedUnit: [mph]
```

Comment:
You rather mean that maxspeed signs are always in mph, right? And AFAIK heights and lengths are (at least on the traffic signs) also mostly in imperial units...

Reply:
Hmm, ok

Reply:
Wikipedia says here: https://en.wikipedia.org/wiki/Road_signs_in_the_United_Kingdom#Units_of_measurement
> The United Kingdom is the only European country to use mostly imperial units rather than SI units on road signs for distance measurements and speed limits, and one of few countries in the world [..] to do so. Vehicle weight limits are signed only in metric (TSRGD 1981), and metric units may be used in addition to imperial ones for height, width and length restrictions.

Just to give a source...

Reply:
Can confirm. Speed limits in UK are always mph.